pkg/ebpf/tracer.go

```diff
@@ -16,20 +16,10 @@ import (
 	"github.com/netobserv/netobserv-ebpf-agent/pkg/flow"
 	"github.com/sirupsen/logrus"
 	"github.com/vishvananda/netlink"
+	"github.com/gavv/monotime"
```

Comment:
Maybe running "go mod vendor" and pushing the new dependencies will fix this compilation error.

Reply:
Done.